mcs/class/System.Web/System.Web.Util/HttpEncoder.cs

```diff
@@ -35,7 +35,7 @@
 using System.Configuration;
 using System.IO;
 using System.Text;
-#if NET_4_0 && !MOBILE
+#if !NO_SYSTEM_WEB_DEPENDENCY && NET_4_0 && !MOBILE
```

Comment:
Rolf suggested making this a different commit if we do this. I'm planning on punting now unless pressed.